data/routes/projectRoute.js

```diff
@@ -0,0 +1,82 @@
+const router = require('express').Router()
+
+const Projects = require('../helpers/projectModel')
+const Actions = require('../helpers/actionModel')
+
+
+
+router.get('/:id', (req, res) => {
+    Projects.get(req.params.id)
```

Comment:
i try to stay away from passing dot notation directly into a function. try to at least console log to check what you are passing OR save it into a variable and pass that